promql/value.go

```diff
@@ -186,12 +186,16 @@ func (m Matrix) ContainsSameLabelset() bool {
 	return false
 }
 
+// Warnings holds evaluation warnings. Currently, it is just a type alias to
```

Comment:
This is not what type aliasing should be used for, using storage.Warnings is fine as we use storage in other parts of the API too.